frappe/__init__.py

```diff
@@ -1522,12 +1521,12 @@ def get_value(*args, **kwargs):
 	"""
 	return db.get_value(*args, **kwargs)
 
-def as_json(obj, indent=1):
+def as_json(obj: Union[Dict, List], indent=1) -> str:
 	from frappe.utils.response import json_handler
-	try:
-		return json.dumps(obj, indent=indent, sort_keys=True, default=json_handler, separators=(',', ': '))
-	except TypeError:
-		return json.dumps(obj, indent=indent, default=json_handler, separators=(',', ': '))
+	obj.pop(None, None)
```

Comment:
A bit conflicted about this. We are mutating the input here. 😅 

It _MIGHT_ make sense to have `None` key in python dictionary.


Why not just disable sorting by default? key order doesn't have any significance in JSON AFAIK. 

We can enable sorting when we are writing doctypes to disk (like fixtures and doctype.json)

Reply:
More weird behaviour 🥲 

<img width="1004" alt="Screenshot 2022-03-18 at 4 00 58 PM" src="https://user-images.githubusercontent.com/9079960/158987122-0d53311a-8e48-413c-806a-960e6e60557b.png">